src/rdb.c

```diff
@@ -1471,6 +1471,7 @@ int rdbSaveBackground(int req, char *filename, rdbSaveInfo *rsi) {
     pid_t childpid;
 
     if (hasActiveChildProcess()) return C_ERR;
+    server.stat_rdb_saves++;
```

Comment:
i notice we only do `server.stat_rdb_saves++` in rdbSaveBackground
and did not do it in SAVE command, (or in FLUSHALL it may trigger rdbsave) (or other)

Reply:
yes, the one in SAVE command is my first thinking, and then I looked at the others (mention in case we forget).
as long as we agree with the stat field meaning, just a random look
i will make a PR with the SAVE one